include/ebpf_nethooks.h

```diff
@@ -107,6 +107,10 @@ bind_hook_t(bind_md_t* context);
 //
 // CGROUP_SOCK_ADDR.
 //
+
+#define VERDICT_REJECT 0
+#define VERDICT_PROCEED 1
```

Comment:
Are these define names something that comes from Linux?   If so, fine.  If not, can we make them more unique, like BPF_SOCK_ADDR_VERDICT_REJECT or something similar?